GameModes/OnlineGameMode.cs

```diff
@@ -22,6 +28,97 @@ public OnlineGameModeType(string value, bool register = false) : base(value, reg
             };
         }
 
+        public static List<string> nonGameplayRemixSettings = new() { "cfgSpeedrunTimer", "cfgHideRainMeterNoThreat", "cfgLoadingScreenTips", "cfgExtraTutorials", "cfgClearerDeathGradients", "cfgShowUnderwaterShortcuts", "cfgBreathTimeVisualIndicator", "cfgCreatureSense", "cfgTickTock", "cfgFastMapReveal", "cfgThreatMusicPulse", "cfgExtraLizardSounds", "cfgQuieterGates", "cfgDisableScreenShake", "cfgHunterBatflyAutograb", "cfgNoMoreTinnitus" };
```

Comment:
unstatic this so gamemodes can change which to sync
also consider a bool or some way to disable sync (e.g. for meadow), personally id make the list nullable